hivemind/dht/validation.py

```diff
@@ -52,3 +53,71 @@ def strip_value(self, record: DHTRecord) -> bytes:
         """
 
         return record.value
+
+    @property
+    def priority(self) -> int:
+        """
+        Defines the order of applying this validator with respect to other validators.
+
+        The validators are applied:
+          - In order of increasing priority for signing a record
+          - In order of decreasing priority for validating and stripping a record
+        """
+
+        return 0
+
+    def merge_with(self, other: 'RecordValidatorBase') -> bool:
```

Comment:
Shouldn't that be an abstract method?

Reply:
I think having this default behavior (a validator refuses to merge with anything and is always applied sequentially) is reasonable for new validators.

Reply:
I have not made it abstract to provide a reasonable default behavior (new validators are applied sequentially by default and refuse to merge with anything).

Reply:
Got it, thanks!